src/viewer.test.tsx

```diff
@@ -80,7 +80,12 @@ describe("SeqViz rendering (React)", () => {
 
     // @ts-expect-error ts-migrate(2322) FIXME: Type '{ file: string; name: string; seq: string; a... Remove this comment to see the full error message
     const wrapper = mount(<SeqViz {...defaultProps} file={fileContents} />);
-    await wrapper.instance().componentDidMount();
+    const _componentDidMount = wrapper.instance().componentDidMount;
```

Comment:
I'm sorry to be pedantic but please no underscore variable names in TS. I tried to turn on a prettier rule to help catch these but ran afoul of some existing variable names, but we should try to reduce over time

https://google.github.io/styleguide/tsguide.html#identifiers